bundles/ui/org.eclipse.smarthome.ui/src/main/java/org/eclipse/smarthome/ui/internal/proxy/ProxyServlet.java

```diff
@@ -51,6 +55,7 @@
  * This servlet also supports data streams, such as a webcam video stream etc.
  *
  * @author Kai Kreuzer - Initial contribution and API
+ * @author Svilen Valkanov - Replaced Apache HttpClient with Jetty
```

Comment:
The changes in this class break the functionality of the proxy servlet.
You can try this with the demo sitemap of openHAB (with the "Multimedia" menu entry).
This includes an image and a video using the proxy servlet, which results in these urls:
http://localhost:8080/proxy?sitemap=demo.sitemap&widgetId=02030100
http://localhost:8080/proxy?sitemap=demo.sitemap&widgetId=02030101

Both result in exceptions like:
```
2016-07-22 15:15:14.572 [WARN ] [e.jetty.servlet.ServletHandler:628  ] - /proxyjava.io.IOException: java.util.concurrent.ExecutionException: java.lang.IllegalArgumentException: Buffering capacity exceeded
	at org.eclipse.smarthome.ui.internal.proxy.ProxyServlet.doGet(ProxyServlet.java:209)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:687)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:790)
	at org.eclipse.equinox.http.servlet.internal.ServletRegistration.service(ServletRegistration.java:61)
	at org.eclipse.equinox.http.servlet.internal.ProxyServlet.processAlias(ProxyServlet.java:128)
	at org.eclipse.equinox.http.servlet.internal.ProxyServlet.service(ProxyServlet.java:60)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:790)
	at org.eclipse.jetty.servlet.ServletHolder.handle(ServletHolder.java:808)
	at org.eclipse.jetty.servlet.ServletHandler.doHandle(ServletHandler.java:587)
	at org.eclipse.jetty.server.session.SessionHandler.doHandle(SessionHandler.java:221)
	at org.eclipse.jetty.server.handler.ContextHandler.doHandle(ContextHandler.java:1127)
Caused by: java.util.concurrent.ExecutionException: java.lang.IllegalArgumentException: Buffering capacity exceeded
	at org.eclipse.jetty.client.util.FutureResponseListener.getResult(FutureResponseListener.java:118)
	at org.eclipse.jetty.client.util.FutureResponseListener.get(FutureResponseListener.java:101)
	at org.eclipse.jetty.client.HttpRequest.send(HttpRequest.java:653)
	at org.eclipse.smarthome.ui.internal.proxy.ProxyServlet.doGet(ProxyServlet.java:199)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:687)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:790)
	at org.eclipse.equinox.http.servlet.internal.ServletRegistration.service(ServletRegistration.java:61)
	at org.eclipse.equinox.http.servlet.internal.ProxyServlet.processAlias(ProxyServlet.java:128)
	at org.eclipse.equinox.http.servlet.internal.ProxyServlet.service(ProxyServlet.java:60)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:790)
	at org.eclipse.jetty.servlet.ServletHolder.handle(ServletHolder.java:808)
Caused by: java.lang.IllegalArgumentException: Buffering capacity exceeded
	at org.eclipse.jetty.client.util.BufferingResponseListener.onHeaders(BufferingResponseListener.java:75)
	at org.eclipse.jetty.client.ResponseNotifier.notifyHeaders(ResponseNotifier.java:103)
	at org.eclipse.jetty.client.ResponseNotifier.notifyHeaders(ResponseNotifier.java:95)
	at org.eclipse.jetty.client.HttpReceiver.responseHeaders(HttpReceiver.java:257)
	at org.eclipse.jetty.client.http.HttpReceiverOverHTTP.headerComplete(HttpReceiverOverHTTP.java:230)
	at org.eclipse.jetty.http.HttpParser.parseHeaders(HttpParser.java:1036)
	at org.eclipse.jetty.http.HttpParser.parseNext(HttpParser.java:1257)
	at org.eclipse.jetty.client.http.HttpReceiverOverHTTP.parse(HttpReceiverOverHTTP.java:156)
	at org.eclipse.jetty.client.http.HttpReceiverOverHTTP.process(HttpReceiverOverHTTP.java:117)
	at org.eclipse.jetty.client.http.HttpReceiverOverHTTP.receive(HttpReceiverOverHTTP.java:69)
	at org.eclipse.jetty.client.http.HttpChannelOverHTTP.receive(HttpChannelOverHTTP.java:89)
```